frameworks/D/hunt/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Clean any files from last run
```

Comment:
There should be something like: `fw_depends mysql dlang` for this to work I guess.

See: https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/D/vibed/setup.sh#L3